sound/smooth_swing_v2.h

```diff
@@ -94,94 +111,123 @@ class SmoothSwingV2 : public SaberBasePassThrough {
         if (speed < smooth_swing_config.SwingStrengthThreshold) {
 #if 1
           if (monitor.ShouldPrint(Monitoring::MonitorSwings)) {
-            STDOUT.print("speed: ");
-            STDOUT.println(speed);
+           STDOUT.print("speed: ");
+           STDOUT.println(speed);
           }
 #endif
           break;
         }
         state_ = SwingState::ON;
         
       case SwingState::ON:
+        //check for AccentSwingThreshold, presence of accent swings and if the accent player is stopped (this prevents clipping)
+        if (speed >=smooth_swing_config.AccentSwingSpeedThreshold && accent_swings_present && !D.isPlaying() && (A.isPlaying() || B.isPlaying()))
+        {
+          while (accentswing == accentrandomizer)
+          {
+           accentswing = random(aswings_);
+          }
+          accentrandomizer = accentswing;
+          STDOUT.print("Accent SWING!");
+          D.set_volume(smooth_swing_config.AccentSwingVolume);
```

Comment:
I think we need to make the volume of the accent dynamic, and change as the swing progresses.
If the swing suddenly stops, the volume of the accent swing should also go down to zero I think.

Reply:
That's a good idea, not quite sure right now how to accomplish that but I'll dig in.

Reply:
Done.  changed up the config variables.
AccentSwingSpeedThreshold - same, deg/sec needed to trigger accent
AccentSwingVolumeSharpness - new, same idea as SwingSharpness.  Used to govern the fade in/out of the swing sound
MaxAccentSwingVolume - renamed from AccentSwingVolume, combines with AccentSwingVolumeSharpness to govern accent swing volume and fade in/out